packages/create-astro/src/templates.ts

```diff
@@ -1,22 +1,5 @@
 export const TEMPLATES = [
-	{
-		title: 'Just the basics (recommended)',
-		value: 'basics',
-	},
-	{
-		title: 'Blog',
-		value: 'blog',
-	},
-	{
-		title: 'Portfolio',
-		value: 'portfolio',
-	},
-	{
-		title: 'Documentation Site',
-		value: 'docs',
-	},
-	{
-		title: 'Empty project',
-		value: 'minimal',
-	},
+	{ value: 'basics', title: 'Include sample files (recommended)' },
```

Comment:
Thinking about this wording a bit. Here was the prompt that we brainstormed in the DX session:

```
How would you like to start your new project?
- Empty
- A few files & folders to help get started
- A personal website + blog starter template
```

And here's what this new prompt looks like:

```
How would you like to start your new project?
  - Include sample files (recommended)
  - Use blog template
  - Empty
```

I like the shorter lines, but this took me longer to parse than I would have expected for a new user prompt. I think it's mainly the use of both "Include" vs. "Use" here (do they mean different things?) but also "Use blog template" and "Include sample files" lack some important context (what sample files? what kind of blog?) Related: It was intentional in the DX session prompt that we call this a personal website and not just a blog.

Bringing these two together, what do you think about this?

```
How would you like to setup your new project?
  - an empty project
  - a few best practices (recommended)
  - a personal website starter kit
```

It still has a bit of the casual wording and context of the first prompt, but with some of the terseness of the second prompt.